### PR TITLE
Update file structure ascii-art

### PR DESCRIPTION
**Pull request by rogeruiz**
Addresses the comments here => https://github.com/18F/web-design-standards/commit/ce138cdbf6c471181c6b3b70e1e14ea18bb3bb53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ First, download the Draft Web Design Standards assets:
 Then, add the `dist` directory files into a relevant place in your code base — likely a directory where you keep third-party libraries:
 
 ```
-- uswds-0.9.0/
-  ├── js
+uswds-0.9.0/
```
Comment (juliaelman) [inline README.md]: Remind me, the version # is not part of the folder name - right?

Comment (rogeruiz) [inline README.md]: The version number is part of the folder and the ZIP archive, but not the filenames any more.

Comment (juliaelman) [inline README.md]: :+1:

**Comment by juliaelman** [comment]
LGTM! :rocket:

**Pull request merged**